BUILD.md

```diff
@@ -74,19 +74,29 @@ Molten-VK compatibility layer
 
 ### On Apple Silicon Macs, Rosetta 2 and the x86_64 version of Homebrew must be used
 
-You can skip this section if you have an Intel Mac. Every time you compile, you need to perform steps 2.
-
-1. `softwareupdate --install-rosetta` # Install Rosetta 2 if you don't have it. This only has to be done once
-2. `arch -x86_64 zsh` # run an x64 shell
+You can skip this section if you have an Intel Mac. Every time you compile, you need to perform step 2.
+
+1. Install Rosetta 2 if you don't have it. This only has to be done once.
+   ```
+   softwareupdate --install-rosetta
+   ```
+2. Run an x86_64 shell.
+   ```
+   arch -x86_64 zsh
+   ```
+3. If you already have brew installed, unload it from your `PATH` so that it doesn't confuse your x86_64-specific installation of brew.
+   ```
+   export PATH=`printf '%s:' $(echo $PATH | tr ':' '\n' | grep -iv "^\/opt\/homebrew\/")`
+   ```
```

Comment:
Fixed in [61ab3a5](https://github.com/cemu-project/Cemu/pull/690/commits/61ab3a5e45e552f74b0729b0bd28a91877ca18d8).

Reply:
Thanks for your fix, but as I said in that comment I think I'd be better to use `plaintext` as the language of these code blocks, since they represent plan simple commands to give in a shell instead of a true bash (or POSIX sh) script

Reply:
Ah OK thanks, I misunderstood. OK, in [9ba3370](https://github.com/cemu-project/Cemu/pull/690/commits/9ba33700222acfb6201e9d14f3166c139fd12279) I've removed the `bash` from the code blocks.

Reply:
I think I haven't been clear again 😅
I suggested you to replace `bash` with `plaintext`, literally, and not to just remove `bash`

Reply:
Ah, OK, fixed in [1688743](https://github.com/cemu-project/Cemu/pull/690/commits/1688743b4cf1c59ed4836bce6fbf2288cf15de7a).